src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -13619,7 +13631,7 @@ void Test23(CL0.CL1 c, Action x23)
             c.VerifyDiagnostics(expected);
         }
 
-        [Fact]
+        [Fact(Skip = "NullableOptOut does not control warnings")]
```

Comment:
I've updated the diagnostics for a couple of these tests, to capture the current expected behavior, and added `// PROTOTYPE` comments to the rest to either update or remove.